setup.py

```diff
@@ -19,6 +19,7 @@ def get_version() -> str:
     "requests",
     "tqdm>=4.42.1",
     "typing-extensions>=3.7.4.3",  # to be able to import TypeAlias
+    "rich>=14.0.0"
```

Comment:
As mentioned in [slack](https://huggingface.slack.com/archives/C02V5EA0A95/p1747847427127569?thread_ts=1747771966.873299&cid=C02V5EA0A95) (private link) I'm quite reluctant to add [rich](https://github.com/Textualize/rich) as a required dependency :confused: